frontend/src/app/modules/matches/matches-computations.js

```diff
@@ -68,3 +77,104 @@ export const computeWinRatio = (playerId, playerMatches) => {
   const wonMatchesCount = playerMatches.filter(match => didPlayerWin(playerId, match)).length
   return (playerMatches.length > 0) ? (wonMatchesCount / playerMatches.length) : 0
 }
+
+export const computeDays = matchChanges => {
+  let worstDayKey = ''
+  let bestDayKey = ''
+  const daysMap = {}
+  for (const match of matchChanges) {
```

Comment:
personally I would do it in two passes for readability - first create a map ratingChangesByDate or smth like that and then compute the max/min of the values. Or maybe I would vote to put a little note how the computation goes in one pass.

Reply:
Done [here](https://github.com/salsita/foosball-rating/pull/114/commits/3d398530cdf75f980e6344e885e4f18dd919311a#diff-08117b849ecc00af7c3c0d609dd86d31)